saleor/graphql/schema.graphql

```diff
@@ -3795,6 +3870,23 @@ type RequestEmailChange {
   accountErrors: [AccountError!]!
 }
 
+type RequestInvoiceCreate {
+  errors: [Error!]!
+  invoice: Invoice
+  invoiceErrors: [InvoiceError!]!
+  order: Order
+}
+
+input RequestInvoiceCreateInput {
+  number: String
```

Comment:
shouldn't that have order also? yeah the number is optional but we also need info which order you are requesting this for...

Reply:
It's in a mutation as a separate argument